### ISSUE TITLE
Minimal setup for tracking only

### ISSUE THREAD

**Issue by Virgile-Dauge** [ValveSoftware/openvr#1459]
Hi, 

I'm currently using SteamVR's impressive tracking capacities for scientific purposes (see [the paper](https://hal.inria.fr/hal-02384704) for further details). Can you give me some insights about the minimal config to use SteamVR (1.14.16) and openvr for tracking only (possibly without UI) ? 
## Current needs 

Ideal functionalities :
* retrieve trackers poses 
* no UI at all (not even vrmonitor if possible, i can get all data i want directly by openvr)
* be able to trigger re-calibration at any time

## Hardware setup 
* 1 or 2 lighthouses v2
* up to 3 trackerv2 (2018)

## Past working solution
I used to be able to run it without HMD, or compositor in the past, simply by changing few settings files and launching vrmonitor process in a python program with Popen on ubuntu 18.04. But since I upgraded to 20.04, VRCompositor is also launched, asking my gpuless station to render a 3D scene.  

1. Changed the following entries in `$STEAM_DIR/config/steamvr.vrsettings`

```json
{
"steamvr" : {
      "activateMultipleDrivers" : true,
      "forcedDriver" : "null",
      "requireHmd" : false,
   },
}
```

2. Changed this entry in `$STEAM_DIR/steamapps/common/SteamVRdrivers/null/resources/settings/default.vrsettings`
```json
{
"driver_null": {
        "enable": true,
   },
}
```

3. Launch Steamvr with runtime in a python script 
```python
steam_runtime = os.path.expanduser('~/.steam/steam/ubuntu12_32/steam-runtime/run.sh')
vr_monitor = os.path.expanduser('~/.steam/steam/steamapps/common/SteamVR/bin/vrmonitor.sh')
proc = subprocess.Popen([steam_runtime, vr_monitor])
```

4. initializing openvr with `VRApplication_Other`

5. Getting poses with  `IRVSystem::getDeviceToAbsoluteTrackingPose`

## Behavior since Ubuntu 20.04 upgrade
VRcompositor window is showing up, together with room setup utility. ( I guess that it wasn't in 18.04 cause vulkan wasn't supported)

## What I tested to avoid this 
I tried launching only vrserver: 
```bash
LD_LIBRARY_PATH=$HOME/.steam/steam/steamapps/common/SteamVR/bin/linux64 ~/.steam/steam/steamapps/common/SteamVR/bin/linux64/vrserver -keepalive
```
But VRcompositor is launched when i initialize openvr with different options : 
* VRApplication_Other
* VRApplication_Background

And i did not succeed to get poses with this `VRApplication_utility` initialization.  

## TL;DR 
How to use openvr (without any UI) to get poses when only vrserver is launched ?

Thanks for opening the nice tool you are supporting :+1:  and maybe for your help too :)

**Comment by Virgile-Dauge** [comment]
Launching the server only works :
```bash
LD_LIBRARY_PATH=$HOME/.steam/steam/steamapps/common/SteamVR/bin/linux64 bash ~/.steam/steam/ubuntu12_32/steam-runtime/run.sh ~/.steam/steam/steamapps/common/SteamVR/bin/linux64/vrserver
```

Then initializing openvr with `VRApplication_Background` launches VRCompositor, which should not be the case for a background app ? 
The IVRSytem still works after killing (`killall -s 9 vrcompositor`) VRCompositor so it's not required for the tracking system to work.

How to prevent vrcompositor launch instead of brutally killing it ?
